agent/fw_wordpress.c

```diff
@@ -285,7 +285,13 @@ static char* nr_wordpress_plugin_from_function(zend_function* func TSRMLS_DC) {
     nrl_verbosedebug(NRL_FRAMEWORK,
                      "Wordpress: cannot determine plugin name:"
                      " missing filename, tag=" NRP_FMT,
-                     NRP_PHP(NRPRG(wordpress_tag)));
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
+     && !defined OVERWRITE_ZEND_EXECUTE_DATA
+                     NRP_PHP((char *)nr_stack_get_top(&NRPRG(wordpress_tags)))
```

Comment:
Is it possible to add unit tests that exercise the functionality of the new tags stack?
https://github.com/newrelic/newrelic-php-agent/blob/main/agent/tests/test_fw_wordpress.c

Reply:
I don't think we should use unit testing here. The manipulation of the stack over the lifetime of a transaction is what we want to be testing. For a unit test to do this, it would need to call the instrumentation functions in order, and at that point it is doing the same thing as the integration tests.

There are already unit tests to verify generic stack functionality.